lib/view_component/compiler.rb

```diff
@@ -237,7 +237,7 @@ def variants_from_inline_calls(calls)
 
     def compiled_template(file_path)
       handler = ActionView::Template.handler_for_extension(File.extname(file_path).gsub(".", ""))
-      template = File.read(file_path)
+      template = File.read(file_path).rstrip
```

Comment:
```suggestion
      template = File.read(file_path)
      template.rstrip!
```

Also minor, but maybe this would result in less allocations?

Reply:
Nice!